easybuild/easyconfigs/u/UMI-tools/UMI-tools-1.0.0-intel-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'UMI-tools'
+version = '1.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://umi-tools.readthedocs.io'
+description = "Tools for handling Unique Molecular Identifiers in NGS data sets"
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('matplotlib', '3.0.0', versionsuffix),
+    ('Pysam', '0.15.1', versionsuffix),
+]
+
+#use_pip = True
```

Comment:
Is there a specific reason we cannot use `pip` here?

Reply:
I guess it didn't work at the time, for some reason, don't recall the details.